cert-manager/cluster-issuer.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+  namespace: kube-system
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: cloud-deployment@forgerock.com
+    # Name of a secret used to store the ACME account private key
+    privateKeySecretRef:
+      name: letsencrypt-prod
```

Comment:
Maybe we should start off with staging, until we know it is really robust

Reply:
Well, I thought about this, and I couldn't really see any point. We're not paying for anything.  And also, I've tested it over and over again and have also got Pavel to test it and never had an issue with the cert request part itself. I can't see we gain anything by using staging except we have an extra step when loading the AM etc in the browser.  We reuse the same cert so we won't get close to any quotas